### ISSUE TITLE
Solaris IPS packages depend on incorrect versions of things

### ISSUE THREAD

**Issue by coderanger** [chef/chef#5456]
The new Solaris IPS packages appear to depend on some package versions that are newer than a standard Solaris 11 box.

```
depend fmri=pkg:/developer/build/make@0.5.11-0.175.2.0.0.34.0 type=require
depend fmri=pkg:/runtime/perl-512@5.12.5-0.175.2.0.0.42.1 type=require
depend fmri=pkg:/shell/bash@4.1.11-0.175.2.0.0.42.1 type=require
depend fmri=pkg:/shell/ksh93@93.21.0.20110208-0.175.2.0.0.42.1 type=require
depend fmri=pkg:/system/core-os@0.5.11-0.175.2.0.0.42.2 type=require
depend fmri=pkg:/system/library/gcc/gcc-c-runtime@4.8.2-0.175.2.0.0.42.1 type=require
depend fmri=pkg:/system/library/math@0.5.11-0.175.2.1.0.4.0 type=require
depend fmri=pkg:/system/library@0.5.11-0.175.2.0.0.42.2 type=require
depend fmri=pkg:/system/linker@0.5.11-0.175.2.0.0.42.2 type=require
```

**Comment by thommay** [comment]
@scotthain @chef/omnibus-maintainers any thoughts?

**Comment by jaymalasinha** [comment]
This was also resolved in chef/omnibus#723 .  It can be closed.
